src/app/containers/StakePage/components/CurrentStakes.tsx

```diff
@@ -194,15 +194,15 @@ function AssetRow(props: AssetProps) {
         </div>
       </td>
       <td className="tw-text-left tw-font-normal">
-        {numberFromWei(props.item[0])} {t(translations.stake.sov)}
+        {weiToFixed(props.item[0], 4)} {t(translations.stake.sov)}
```

Comment:
It looks good now, just use `weiTo4` function for every file in this PR.

Reply:
yeah, its done